Job for sssd.service failed because the control process exited with error code.
*
Traceback (most recent call last):
  File "test/verify/check-static-login", line *, in testClientCertAuthentication
    m.execute("systemctl restart sssd")
